converter/service/converter.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+import gen_universe
+import json
+import logging
+import os
+import re
+
+from enum import Enum
+from http import HTTPStatus
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from urllib.error import URLError, HTTPError
+from urllib.parse import parse_qsl, urlparse
+from urllib.request import Request, urlopen
+
+
+# Binds to all available interfaces
+HOST_NAME = ''
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT_UNIVERSECONVERTER'])
```

Comment:
so it seems that marathon provides the port with `PORT_<whatever name you specify in the marathon app json portDefinitions field>`. We can use `PORT0` or `PORT_UNIVERSECONVERTER`